Rocket.Chat/Views/Cells/Chat/ChatMessageCell.swift

```diff
@@ -272,3 +272,31 @@ extension ChatMessageCell: UIGestureRecognizerDelegate {
     }
 
 }
+
+// MARK: Accessibility
+
+extension ChatMessageCell {
+    override func awakeFromNib() {
+        isAccessibilityElement = true
+    }
+
+    override var accessibilityIdentifier: String? {
+        get { return "message" }
+        set { }
+    }
+
+    override var accessibilityLabel: String? {
+        get { return message?.accessibilityLabel }
+        set { }
+    }
+
+    override var accessibilityValue: String? {
+        get { return message?.accessibilityValue }
+        set { }
+    }
+
+    override var accessibilityHint: String? {
+        get { return "Double tap and hold for actions" }
```

Comment:
Want to use i18n here?

Reply:
VoiceOver already says "actions available" by default, so I was going to remove it and forgot 🤦‍♂️